src/Symfony/Component/ObjectMapper/Tests/ObjectMapperTest.php

```diff
@@ -100,7 +100,7 @@ public function testHasNothingToMapTo()
     {
         $this->expectException(MappingException::class);
         $this->expectExceptionMessage('Mapping target not found for source "class@anonymous".');
-        (new ObjectMapper())->map(new class () {});
+        (new ObjectMapper())->map(new class {});
```

Comment:
Shall we enable:
https://cs.symfony.com/doc/rules/operator/new_with_parentheses.html#example-2